src/plugins/Transloadit/index.js

```diff
@@ -700,12 +705,14 @@ module.exports = class Transloadit extends Plugin {
       this.uppy.on('transloadit:complete', onAssemblyFinished)
       this.uppy.on('transloadit:assembly-error', onAssemblyError)
       this.uppy.on('transloadit:import-error', onImportError)
-    }).then(() => {
+    }).then((result) => {
```

Comment:
So still returning the result, but not using it?

Reply:
oops. `result` is always undefined there, should remove that. thanks!